ui/app/clinical/common/controllers/visitController.js

```diff
@@ -11,12 +11,39 @@ angular.module('bahmni.clinical')
                 $state.current.views['print-content'].templateUrl;
             var showProviderInfo = appService.getAppDescriptor().getConfigValue('showProviderInfoinVisits');
             $scope.showProviderInfo = showProviderInfo !== false ? true : showProviderInfo;
+            var showPatientInfo = appService.getAppDescriptor().getConfigValue('showPatientInfoInVisits');
+            $scope.showPatientInformation = showPatientInfo !== false ? true : showPatientInfo;
```

Comment:
@Arjun-Go @kavitha-sundararajan The ternary operator used here is redundant. If showPatientInfo is false, it sets `$scope.showPatientInformation` to true, which might not be the intended behavior. Please validate the approach once and maybe try - `$scope.showPatientInformation = showPatientInfo !== false;` which simplifies the initialization further. Still if showPatientInfo is `undefined, null, or true, $scope.showPatientInformation` will be true. Do confirm if this is the intended behaviour.